build/x86_64_centos6/Dockerfile

```diff
@@ -19,6 +19,9 @@ RUN yum -y update \
         ruby \
         which \
         zlib-devel \
+ # CentOS 6 comes with git 1.7.1, and omnibus requires at least 1.7.2.
```

Comment:
[Optional] Do we want to remove the `git` above?

Reply:
I think it makes sense to keep the original installation and only upgrade from rpmforge. Changed install to upgrade to make that obvious. PTAL.

Reply:
SG